server.R

```diff
@@ -15,7 +15,7 @@ shinyServer(function(input, output) {
     coral_health_plot_caption <- reactive({
         generate_coral_health_caption(input)
     })
-    output$coral_health_plot <- renderPlot({
+    coral_health_plot <- reactive({
```

Comment:
Good solution - this works well, and the app still runs fast with the reactive expression